src/crystal/system/unix/getrandom.cr

```diff
@@ -26,7 +26,7 @@ module Crystal::System::Random
   private def self.init
     @@initialized = true
```

Comment:
Hmm. This is a tangent to this PR, but looking at this method and `random_bytes` make me wonder, is this safe for multicore operation? If two methods call `random_bytes` at almost the same point, it may reach a point where `@@initialized` is set to true, but neither `@@getrandom_available` or `@@urandom` is set.

Is there anything that make certain `init` is called in the prelude before the scheduler is set up (which would be one way to avoid this)?

(comment applies only to `mt_preview`)

Reply:
I'll give you a bad answer: Currently the Crystal::Hasher's seed is initialized before any thread is created. So this scenario will never actually happen.

Reply:
Well, that is certainly a lot better than it could have been. I wonder how many hidden race conditions lurk in stdlib due to class variables being so prone to unsafety on initialization.